index.html

```diff
@@ -9,76 +9,259 @@
+              <div id="card_secondary"><p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p></div>
+              <ul id="card_tags">
+                <li class="card_btns"><card_btns_font class="card_btns_font">html</card_btns_font></li>
+                  <li class="card_btns"><card_btns_font class="card_btns_font">bootstrap</card_btns_font></li>
+                  <li class="card_btns"><card_btns_font class="card_btns_font">Ruby</card_btns_font></li>
+                  </ul>
+            </li>
+          </div>
+        </ul>
+      </div>
+
+    <div id="about_me">
+      <div><p id="me_primary_text">About me</p></div>
+      <div><p id="me_secondary_text">Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p></div>
+      <a href="https://github.com/glitcher255/micro-porfolio"><me_btn class="orange_btns" id="me_btn"><p id="me_btn_text">Get My Resume</p></me_btn></a>
+      <div id="me_art"></div>
+      <div id="me_rectangle"><img src="./assets/Rectangle 73.png" alt="Rectangle"></div>
+      <div id="me_vector"><img src="./assets/Vector 4.png" alt="Divider"></div>
+    </div>
+    
+
+    <div id="skill">
+
+      <ul>
+      <li id="frame30">
+        <div id="rectangle_rectangle"></div><p id="language_text">language</p>
+        <ul>
+        <li class="frame30_text"><p class="frame30_font">Javascript</p></li>
+        <li class="frame30_text"><p class="frame30_font">Ruby</p></li>
+        <li class="frame30_text"><p class="frame30_font">Html</p></li>
+        <li class="frame30_text"><p class="frame30_font">Css</p></li>
+      </ul>
+  
+      <li id="frame30">
+        <div id="square_rectangle"></div><p id="language_text">Frameworks</p>
+        <ul>
+        <li class="frame30_text"><p class="frame30_font">Bootstrap</p></li>
+        <li class="frame30_text"><p class="frame30_font">Ruby on Rails</p></li>
+        <li class="frame30_text"><p class="frame30_font">RSPec</p></li>
+        <li class="frame30_text"><p class="frame30_font">CapyBara</p></li>
+        <li class="frame30_text"><p class="frame30_font">Selenium</p></li>
+        </ul>
+  
+      <li id="frame30">
+        <div id="circle_rectangle"></div><p id="language_text">Skills</p>
+        <ul>
+        <li class="frame30_text"><p class="frame30_font">
+        <li class="frame30_text"><p class="frame30_font">Codekit</p></li>
+        <li class="frame30_text"><p class="frame30_font">Github</p></li>
+        <li class="frame30_text"><p class="frame30_font">Codepen</p></li>
+        <li class="frame30_text"><p class="frame30_font">Gitlab</p></li>
+        <li class="frame30_text"><p class="frame30_font">Terminal</p></li>
+        </ul>
+    </ul>
+    </div>
+</recentwork>
```

Comment:
Kindly, in previous reviews I told you only use valid HTML labels, and turn them into correct or divs, I attached documentation about HTML and its correct uses. (Check all your code to remove them all) https://www.tutorialspoint.com/html5/html5_tags.htm